CCMKTGGP/Away
# Yousuf billing

**Title:** Integrate Plan Details API and added enabled it on accounts page as well 

**Description:**
This PR includes the following changes:

- Added `PlanDetails` component to fetch and display plan details from `/api/get-plan-details`.
- Used Flexbox to display `PlanCard` components horizontally.
- Styled `PlanCard` components with Tailwind CSS according to design specifications.

**How to Test:**
1. Navigate to the "billingInfo" tab.
2. Verify correct API data fetching and horizontal display of `PlanCard` components.

Please review and let me know if there are any questions. Thank you!
